mountpoint-s3-crt-sys/Cargo.toml

```diff
@@ -9,6 +9,38 @@ edition = "2021"
 license = "Apache-2.0"
 repository = "https://github.com/awslabs/mountpoint-s3"
 description = "Rust FFI bindings to the AWS Common Runtime for Mountpoint for Amazon S3."
+exclude = [
+    # Exclude large files/directories not required to build the CRT (e.g. tests, docs)
+    "crt/aws-c-auth/tests/*",
```

Comment:
Maybe something like this instead of manually excluding just some tests (except a couple we need to keep for feature detection):
```
    "crt/*/tests",
    "!crt/aws-lc/tests/compiler_features_tests",
    "!crt/s2n-tls/tests/features",
    "crt/*/docs",
```